_overviews/compiler-options/index.md

```diff
@@ -44,34 +40,41 @@ This page comes to the rescue for the people to find&hellip;
 ```bash
 scalac [ <options> ] <source files>
 ```
+Boolean flags are specified in the usual way:
+
+`scalac -Werror -Xlint Hello.scala`
+
+Options that require arguments use "colon" syntax:
+
+`scalac -Vprint:parser,typer`
 
-E.g. `scalac -encoding utf8 -Xfatal-warnings Hello.scala`
+Options that take just a single argument accept traditional syntax:
 
-Default paths can be listed by running a command line tool:
+`scalac -d /tmp`
+
+Default paths can be listed by running a tool in the distribution:
 ```
 scala scala.tools.util.PathResolver [ <options> ]
 ```
 
```

Comment:
This does not work for me:
```
$ scala scala.tools.util.PathResolver
Ignoring spurious arguments: scala.tools.util.PathResolver
Welcome to Scala 3.1.3 (17.0.3, Java OpenJDK 64-Bit Server VM).
Type in expressions for evaluation. Or try :help.
                                                                                                                               
scala> 
```

Reply:
Am I doing something wrong?

Reply:
It's scala 2, `scala.tools` vs `dotty.tools`. Worth mentioning.